src/apps/sketch/index.jsx

```diff
@@ -46,6 +46,7 @@ const Sketch = () => {
     }
 
     const startBoard = () => {
+        dataChannel.current.getChannel('meta').send('active_board');
```

Comment:
store channel in useRef and use it to manage all coms

Reply:
sure